todo_app/app.py

```diff
@@ -1,14 +1,22 @@
-from flask import Flask
-
+from flask import Flask, render_template, request
+from todo_app.data.session_items import *
 from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
 
-@app.route('/')
+@app.route('/', methods = ['POST', 'GET'])
 def index():
-    return 'Hello World!'
+    if request.method == 'POST':
+        itemAdded = request.form['addTo']
+        add_item(itemAdded)
+        return render_template('index.html', lstToDo = get_items())
+        
+
+
+    else:
+        return render_template('index.html')
```

Comment:
You should split this index function into two functions with different `@app.route` decorators.

Your GET currently isn't setting `lstToDo = get_items()` but should be.

Your POST should actually `return redirect('/')` to make the browser send a new GET request to reload the page. This means:
- No repeated code
- The browser will show the correct URL no matter what your POST endpoint's route is
- The user can refresh the page without resubmitting the form.